pkg/metric/metric.go

```diff
@@ -10,28 +10,14 @@ var (
 	HttpDuration = promauto.NewHistogramVec(prometheus.HistogramOpts{
 		Name: "search_http_duration_seconds",
 		Help: "Latency of of HTTP requests in seconds.",
-	}, []string{"path", "method"})
+	}, []string{"code", "query_type"})
```

Comment:
Remove `query_type` because this PR doesn't implement it yet.
```suggestion
	}, []string{"code"})
```